### DISCUSSION TITLE
[Feature Request] Make Walking Route visible at low zoom

### DISCUSSION THREAD

**Discussion by letypequividelespoubelles** [osmandapp/OsmAnd#11559] [Ideas]
# 🚀 feature request

### Description

Many [walking  route](https://wiki.openstreetmap.org/wiki/Walking_Routes) are in OSM but it's not possible to see them in low zoom, even in hiking profile (topo map). They are visible up to zoom 11 and disapear at zoom 10. It would be very usefull to be able to see them up to zoom 6 or 5 (to be discussed maybe). It would be a great help while preparing one month hiking.

### Describe alternatives you've considered

What I'm doing now : dl gpx track of the walking route and import them in OSMAnd to be able to see them at any zoom. Not very convenient, and you can't do it while hiking without network.

**Comment by letypequividelespoubelles** [comment]
If you show me where it is in the code, I may give it a try

**Comment by thompehw** [comment]
I had the same wishes, and made a custom renderer that meets my personal requirements.  You can take a look at it on the old OsmAnd google group:
[https://groups.google.com/g/osmand/c/WyGJS0Dekrk/](https://groups.google.com/g/osmand/c/WyGJS0Dekrk/) 

Scroll down through the thread, as there are illustrations and updates. I expect it's not quite coded correctly, but I've been using it for years without problems.

**Comment by sonora** [comment]
If you use OsmAnd's Touring view Map style, it should have no limitations whatever, it displays tracks at all zoom levels (I heard of users using this on the Jacob Trail  ;) ).

**Comment by scaidermern** [comment]
> If you use OsmAnd's Touring view Map style, it should have no limitations whatever, it displays tracks at all zoom levels (I heard of users using this on the Jacob Trail ;) ).

I think the question is about _hiking routes_, not about GPX files. Even with the touring style, they are only visible up to zoom level 12 and disappear with zoom level 11 (where the world map starts).

**Comment by sonora** [comment]
Right, that data is not contained in the basemap, hence cannot be displayed. The only workaround I can think of for now is to create a gpx for your itinerary, then display that.

**Comment by letypequividelespoubelles** [comment]
@sonora I don't undersatnd your point. The data is in OSM (see for exemple [HRP](https://www.openstreetmap.org/relation/2018553)) and visible in OSMAnd, but only in high zoom (up to 11). My issue is that I would like it to be displayed even at low zoom (6 or 5 maybe)

**Comment by sonora** [comment]
@letypequividelespoubelles Yes, understood. My point was the data is not in our OsmAnd preparation of the world basemap, hence it is not trivial to display.

**Comment by letypequividelespoubelles** [comment]
@sonora I understand your point, but it's not displayed even at zoom where local maps are used. Maybe we can display walking route while local maps are used, which is not not the case now.
@vshcherb Can you reopen the issue please ? Thanks
